ztp/kube-compare-reference/optional/storage/StorageLVMSubscriptionNS.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-storage
+  labels:
+    workload.openshift.io/allowed: "management"
```

Comment:
We should add a comment before this line saying that from 4.15+ LVM operator can be part of the management partition.